src/controller/CHIPCluster.h

```diff
@@ -93,6 +95,14 @@ class DLL_EXPORT ClusterBase
     template <typename AttrType>
     CHIP_ERROR WriteAttribute(const AttrType & requestData, void * context, ClusterId clusterId, AttributeId attributeId,
                               WriteResponseSuccessCallback successCb, WriteResponseFailureCallback failureCb)
+    {
+        return WriteAttribute(requestData, context, clusterId, attributeId, successCb, failureCb, nullptr);
```

Comment:
nit: it may not be immediately obvious what nullptr is here. maybe `, nullptr /* doneCb */);` would make it easier to read

Reply:
done